client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -27,7 +27,7 @@ const ConnectedSingleSiteWpcom = connectOptions( ( props ) => {
 					plan={ PLAN_PREMIUM }
 					customerType="business"
 					className="themes__showcase-banner"
-					title={ translate( 'Upload your own themes with our Premium and Business plans!' ) }
+					title={ translate( 'Upload your own themes with our Business and eCommerce plans!' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.